src/hooks/useDomainInfo.tsx

```diff
@@ -138,6 +140,7 @@ export default function useDomainInfo({
       ticker,
       owner,
       controllers,
+      logo: state.Logo ?? '',
```

Comment:
Double checking, is this supposed to be capitalized "Logo" and not .logo ?

Reply:
yup